admin/client/src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+import { Tab, Nav, NavItem } from 'react-bootstrap-ss';
+
+class Tabs extends SilverStripeComponent {
+  /**
+   * Determines a default tab to be opened and validates the given default tab.
```

Comment:
oh, but the child Tab would also have its own `getDefaultActiveKey()` and `activeKey` property.

I think it would lose its self-contained nature if we're extending to that level of control for `defaultActiveKey`/`activeKey`.